### ISSUE TITLE
ffmpeg.py _load_infos failing. 'Reader' object has no attribute '_frame_catcher'

### ISSUE THREAD

**Issue by ekcheng** [imageio/imageio#251]
I'm intermittently getting the following error when calling `get_reader` to open a movie file:

```
10854 Traceback (most recent call last):
10859   File "/home/ubuntu/redactor/worker/../core/reader.py", line 53, in __init__
10860     self.__reader = imageio.get_reader(moviefile, format='FFMPEG')
10861   File "/home/ubuntu/redactor/venv/local/lib/python2.7/site-packages/imageio/core/functions.py", line 111, in get_reader
10862     return format.get_reader(request)
10863   File "/home/ubuntu/redactor/venv/local/lib/python2.7/site-packages/imageio/core/format.py", line 160, in get_reader
10864     return self.Reader(self, request)
10865   File "/home/ubuntu/redactor/venv/local/lib/python2.7/site-packages/imageio/core/format.py", line 209, in __init__
10866     self._open(**self.request.kwargs.copy())
10867   File "/home/ubuntu/redactor/venv/local/lib/python2.7/site-packages/imageio/plugins/ffmpeg.py", line 301, in _open
10868     self._load_infos()
10869   File "/home/ubuntu/redactor/venv/local/lib/python2.7/site-packages/imageio/plugins/ffmpeg.py", line 449, in _load_infos
10870     self._terminate()
10871   File "/home/ubuntu/redactor/venv/local/lib/python2.7/site-packages/imageio/plugins/ffmpeg.py", line 421, in _terminate
10872     if self._frame_catcher:
10873 AttributeError: 'Reader' object has no attribute '_frame_catcher'
```

From what I can tell, it looks like `_load_infos` is failing in [`ffmpeg.py`](https://github.com/imageio/imageio/blob/master/imageio/plugins/ffmpeg.py#L433).

I am running `imageio` v2.1.1 as part of an upstart script on Ubuntu 14.04 on an AWS EC2 instance. I noticed there is a sleep routine for a few seconds prior to this in `_load_infos`. Is it possible that it just needs to wait longer since it's running on a potentially slow virtualized environment on AWS?

Any advice on the root cause would be greatly appreciated. Thanks.

**Comment by almarklein** [comment]
Duplicate of #235, fixed in #236, but that has not made it into a release yet. I plan to do a release real soon.

**Comment by ekcheng** [comment]
Thanks, @almarklein! Sorry I didn't catch those before posting. Look forward to the next release.
